package.json

```diff
@@ -43,7 +43,7 @@
 	},
 	"engines": {
 		"npm": ">=7.0.0",
-		"node": ">=14.0.0"
+		"node": ">=14.x <16.x"
```

Comment:
I _think_ [this syntax](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines) will work?

Reply:
yea! https://jubianchi.github.io/semver-check/#/%3E%3D14.x%20%3C16.x/16.13.0